mappings/net/minecraft/text/MutableText.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_5250 net/minecraft/text/MutableText
+	COMMENT A text with mutation operations.
+	COMMENT
+	COMMENT @see Text
```

Comment:
```suggestion
	COMMENT A {@link Text} with mutation operations.
```

Reply:
The text is already available as this class extends Text. also won't use embed link to replace see because see tag is more outstanding while embed links tend to get ignored.

Reply:
If `MutableText` already extends `Text`, why link at all? `extends Text` will be right below in code and generated JD should have an automatic link too.